package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "ipfs-merkle-dag",
```

Comment:
don't forget to deprecate the latest versions of these modules on npm with a message that they were renamed